connectors/aws-cert/src/test/resources/awscert/responses/success/cards/single-card.json

```diff
@@ -79,7 +79,8 @@
       ],
       "image": {
         "href": "https://my-connector/images/connector.png"
-      }
+      },
```

Comment:
Card action url in JiraConnector is substituted with mock provider host and port. So the hash value was getting changed every time when we run the test case. I am replacing the actual has h value with ```test-hash``` before comparing the json value.